### ISSUE TITLE
ImportError: cannot import name 'textwriter' from 'botocore.docs.bcdoc' (Python3.7.7)

### ISSUE THREAD

**Issue by EdgarMesa** [awslabs/aws-shell#247]
It lets me pip install aws-shell correctly but when I try to run the shell it keeps giving me the same import error (even for different python versions)

**Comment by davidvivianoASICS** [comment]
I am having a similar issue but it is thrown during the handling of another exception:
`ADIs-MacBook-Pro-37:~ adiloaner$ aws-shell
First run, creating autocomplete index...
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/utils.py", line 84, in file_contents
    with open(filename, mode) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/Users/adiloaner/.aws/shell/cache/completions-1.18.142.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/index/completion.py", line 63, in load_index
    contents = self._fslayer.file_contents(filename)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/utils.py", line 87, in file_contents
    raise FileReadError(str(e))
awsshell.utils.FileReadError: [Errno 2] No such file or directory: '/Users/adiloaner/.aws/shell/cache/completions-1.18.142.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/__init__.py", line 39, in main
    index_str = indexer.load_index(utils.AWSCLI_VERSION)
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/index/completion.py", line 65, in load_index
    raise IndexLoadError(str(e))
awsshell.index.completion.IndexLoadError: [Errno 2] No such file or directory: '/Users/adiloaner/.aws/shell/cache/completions-1.18.142.json'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/3.8/bin/aws-shell", line 8, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/__init__.py", line 43, in main
    from awsshell.makeindex import write_index
  File "/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/awsshell/makeindex.py", line 8, in <module>
    from botocore.docs.bcdoc import textwriter
ImportError: cannot import name 'textwriter' from 'botocore.docs.bcdoc' (/Library/Frameworks/Python.framework/Versions/3.8/lib/python3.8/site-packages/botocore/docs/bcdoc/__init__.py)`

**Comment by pybeaudouin** [comment]
Confirmed with Python 3.8 also. This seems related to aws-cli and boto3:
* https://github.com/aws/aws-cli/pull/5538
* https://github.com/boto/boto3/issues/2596

**Comment by EdgarMesa** [comment]
![Err](https://user-images.githubusercontent.com/60709853/93837565-7fe4c780-fc7e-11ea-8275-381caa4cc566.PNG)
Same error for Python3.8.1

**Comment by gknapp** [comment]
Also seeing this issue on Python 3.8, Windows 10:

```
$ pip freeze
aws-shell==0.2.1
awscli==1.18.144
boto3==1.15.3
botocore==1.18.3
colorama==0.4.3
configobj==5.0.6
docutils==0.15.2
jmespath==0.10.0
prompt-toolkit==1.0.18
pyasn1==0.4.8
Pygments==2.7.1
python-dateutil==2.8.1
PyYAML==5.3.1
rsa==4.5
s3transfer==0.3.3
six==1.15.0
urllib3==1.25.10
wcwidth==0.2.5
```

**Comment by rymdninja** [comment]
Same here, using the latest builds of aws-shell and aws-cli from the repo.

$ pip freeze
astroid==2.4.2
aws-shell @ git+https://github.com/awslabs/aws-shell.git@8950f03d9d720879890af6c11537b8f9789ce5a9
awscli @ git+https://github.com/aws/aws-cli@dabd57baa645ac3e56f39ffcc0e8bec76ccfd50a
boto3==1.15.8
botocore==1.18.8
certifi==2020.6.20
chardet==3.0.4
colorama==0.4.3
configobj==5.0.6
docutils==0.15.2
idna==2.10
isort==4.3.21
jmespath==0.10.0
lazy-object-proxy==1.4.3
mccabe==0.6.1
prompt-toolkit==1.0.18
pyasn1==0.4.8
Pygments==2.7.1
pylint==2.5.3
python-dateutil==2.8.1
PyYAML==5.3.1
requests==2.24.0
rsa==4.5
s3transfer==0.3.3
six==1.15.0
toml==0.10.1
urllib3==1.25.9
wcwidth==0.2.5
wrapt==1.12.1

**Comment by nthienan** [comment]
Experiencing the same error

**Comment by claick-oliveira** [comment]
> Experiencing the same error

+1

**Comment by joguSD** [comment]
Confirming this as an issue. I've opened a fix in: #250 

As a temporary working around, you'll need to pin below `aws-cli<1.18.140` and `boto3<1.15.0`.
